jwst/datamodels/__init__.py

```diff
@@ -41,6 +41,7 @@
 from .model_base import DataModel
 from .amilg import AmiLgModel
 from .asn import AsnModel
+from .barshadow import BarshadowModel
```

Comment:
You should add `BarShadowModel` to `__all__` below as well.

Reply:
And add it in the proper alphabetical order!